app/src/main/java/com/capstone/weatherapp/CityListViewModel.kt

```diff
@@ -3,5 +3,5 @@ package com.capstone.weatherapp
 import androidx.lifecycle.ViewModel
 
 class CityListViewModel : ViewModel() {
-    // TODO: Implement the ViewModel
+    var arrayList = ArrayList<Int>(List(100) { it + 1 })
```

Comment:
let's add a `private var _list` and then a `public val list` that can be used for fetching this data instead of just having one `arrayList` variable